webapp/.eslintrc.json

```diff
@@ -32,7 +32,10 @@
     "beforeEach": true
   },
   "settings": {
-    "import/resolver": "webpack"
+    "import/resolver": "webpack",
+    "react": {
+      "version": "detect"
```

Comment:
Should this change also go into https://github.com/mattermost/mattermost-plugin-starter-template?

Reply:
I would say it should. It mainly remove a warning during the linting.